packages/edit-site/src/components/block-editor/index.js

```diff
@@ -47,31 +48,57 @@ const LAYOUT = {
 };
 
 const NAVIGATION_SIDEBAR_NAME = 'edit-site/navigation-menu';
+const NavMenuSidebarToggle = () => {
+	const { isNavigationSidebarOpen } = useSelect( ( select ) => {
+		return {
+			isNavigationSidebarOpen:
+				select( interfaceStore ).getActiveComplementaryArea(
```

Comment:
I think we can not have two complementary areas active at the same time, so, for example, we can not have the global styles sidebar active in the view mode. Maybe that's ok, but if not, I guess we can simply not use the interface package.

Reply:
This is basically the same code as trunk, I just moved it to a dedicated component for clarity. I'm not entirely certain what you're suggesting here :)

Reply:
I think it's ok, the navigation sidebar is still part of the one complementary area in this PR (as it stands). These are the three items:
![Screen Shot 2022-10-14 at 3 33 17 pm](https://user-images.githubusercontent.com/677833/195790662-28127f60-9243-4133-b602-82f85e8b0915.png)

I think the plan was to move it to the left side of the screen, so maybe that's where the confusion is. If/when that happens this would have to change.